lib/signaling/v3/remoteparticipant.js

```diff
@@ -24,11 +25,13 @@ class RemoteParticipantV3 extends RemoteParticipantV2 {
     setPriority,
     setRenderHint,
     clearTrackHint,
+    getTrackStats,
```

Comment:
this parameters keep increasing. Please change them to use object destructuring, this makes it much readable at caller site. So instead of 
```
 constructor(
    participantState,
    getPendingTrackReceiver,
    getInitialTrackSwitchOffState,
    setPriority,
    setRenderHint,
    clearTrackHint,
    getTrackStats,
    options
)
```

we should change to:
```
 constructor({
    participantState,
    getPendingTrackReceiver,
    getInitialTrackSwitchOffState,
    setPriority,
    setRenderHint,
    clearTrackHint,
    getTrackStats,
    options
})
```

Reply:
@makarandp0 It is tedious, but we typically have arguments for properties that are required, and all optional properties in options. So, I don't want to change it just yet. It will also result in changes elsewhere, thereby making the PR more difficult to review.